modules/php/Actions/CircleNextInRow.php

```diff
@@ -59,7 +59,7 @@ public function actCircleNextInRow()
           Notifications::scribbleAstronaut($player, $scribble) :
           Notifications::scribblePlanning($player, $scribble);
         $jokers = $args['jokers'] ?? [];
-        if (!empty($jokers) && in_array($slot, array_keys($jokers))) {
+        if (in_array($slot, array_keys($jokers))) {
```

Comment:
Proposal for this kind of situation (personnal taste) :
$slotId = $jokers[$slot] ?? null;
if(!is_null($slotId)){
    $scribble = $scoresheet->addScribble($slotId, SCRIBBLE_CIRCLE);
    ...
}

Reply:
Yeah, that makes sense